src/components/header.tsx

```diff
@@ -56,7 +56,7 @@ const Header = ({ darkModeController }: Props) => (
         <button
           type="button"
           className="dark-mode-toggle"
-          onClick={() => {
+          onClick={(): void => {
```

Comment:
@benhalverson Can we work on eslint together to avoid needing the obvious coercion… I know it is an acquired taste but `onClick` should be able to throw if my function is doing something wrong, and by forcing things we basically saturate things and fail to catch what matters.